config/hathitrust_config.rb

```diff
@@ -50,20 +43,24 @@ def filter
         }
       end
 
+      def profiles
+        subsets + [superset]
+      end
+
       private
 
       def subset_volume_repo
-        @rights_volume_repo ||= RightsVolumeRepo.new(rights_db_connection)
+        @rights_volume_repo ||= Repository::RightsVolumeRepo.new(db_connection)
       end
 
       def superset_volume_repo
```

Comment:
The naming of this and subset_volume_repo caused me a bit of confusion, but the functionality of it appear sound.  Refactor this naming later.